test/core/test_sympy_python_convert.py

```diff
@@ -164,13 +204,22 @@ def testConstant(self):
     def testGamma(self):
         self.compare(
             Expression(SymbolGamma, Symbol("Global`z")),
+<<<<<<< HEAD
             sympy.gamma(sympy.Symbol(f"{sympy_symbol_prefix}Global`z")),
+=======
+            sympy.gamma(sympy.Symbol("_m3g`z")),
+>>>>>>> 999c7a5d (WIP - trace sympy calls)
         )
         self.compare(
             Expression(SymbolGamma, Symbol("Global`z"), Symbol("Global`x")),
             sympy.uppergamma(
+<<<<<<< HEAD
```

Comment:
That was later addressed using the version in master without any changes.

Because there are substantive changes now in the failure, I suspect there is a deeper problem and something that needs conversion (or we need to use the prefix variables substitutions more).